Gruntfile.js

```diff
@@ -189,10 +192,11 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-coveralls');
   grunt.loadNpmTasks('grunt-contrib-connect');
   grunt.loadNpmTasks('grunt-umd');
-  grunt.loadNpmTasks("grunt-jscs");
+  grunt.loadNpmTasks('grunt-jscs');
+  grunt.loadNpmTasks('grunt-contrib-clean');
 
   // Default task.
-  grunt.registerTask('default', ['jshint', 'jscs', 'test', 'coverage', 'concat',
+  grunt.registerTask('default', ['clean', 'jshint', 'jscs', 'test', 'coverage', 'concat',
     'umd', 'uglify']);
 
```

Comment:
Nevermind I'll merge it now and make the change myself so I can continue development.